source/unified-test-format/tests/invalid/expectedError-isTimeoutError-type.yml

```diff
@@ -0,0 +1,15 @@
+description: "expectedError-isTimeoutError-type"
+
+schemaVersion: "1.0"
```

Comment:
I noticed you used `schemaVersion: "1.5"` in `initialCollectionData-additionalProperties.yml` below. Did you also intend to use that here since `isTimeoutError` is new syntax?

I realize it makes no difference, since this file is validated against the most recent schema anyway -- but if that ever changes it'd be helpful to have them be in sync.